### PR TITLE
spec: typos

### PR DESCRIPTION
**Pull request by decentralisedkev**

### CODE REVIEW

File: spec/ZkVM.md [hunk 1]
```diff
@@ -1232,15 +1232,15 @@ This is different from [`var`](#var) in that the variable is not represented by
 
 Pushes an [expression](#expression-type) `expr` corresponding to the [minimum time bound](#time-bounds) of the transaction.
 
-The one-term expression represents time bound as a weight on the R1CS constant `1` (see [`cost`](#const)).
+The one-term expression represents time bound as a weight on the R1CS constant `1` (see [`const`](#const)).
```
Comment (oleganza) [inline spec/ZkVM.md]: thanks!

File: spec/ZkVM.md [hunk 2]
```diff
@@ -1669,7 +1669,7 @@ the second-from-the-top is not a [contract](#contract-type).
 
 _contract(P) L R_ **left** → _contract(L)_
 
-1. Pops the right [predicate](#predicate) `R`, then the left [predicate](#predicate) `L`.
+1. Pops the right [predicate](#predicate) `R`, then the left [predicate](#predicate) `L` and a [contract](#contract-type) `contract`.
```
Comment (oleganza) [inline spec/ZkVM.md]: if you spec to pop the contract, then we need to put it back on the stack on step 4. In both instructions.

Comment (decentralisedkev) [reply spec/ZkVM.md]: Step5 for left says: "Replaces the contract’s predicate with L and leaves the contract on stack." Could we change this to ""Replaces the contract’s predicate with L and puts the contract back on stack." ?

Comment (oleganza) [reply spec/ZkVM.md]: Yes, let's do that. Thank you.

Comment (oleganza) [reply spec/ZkVM.md]: s/puts the contract/pushes the contract/, I'd like to consistently use "pop", "push" terms for the stack operations.

**Pull request merged**